docs/wiki/installation/install-windows.md

```diff
@@ -1,13 +1,69 @@
 As of osquery 1.8.2+ the Windows builds are feature-complete but provide a limited set of tables compared to macOS and Linux.
 
-## Chocolatey
+## Installing osquery on Windows
+We recommend installing Windows via the Chocolatey package system however a helper script for generating an **MSI** installer is available at `tools\deployment\make_windows_package.ps1` by invoking with an 'msi' parameter. Further manual installation instructions are detailed below for those needing more custom deployment options.
```

Comment:
I'd recommend keeping double-newline between header and body. Some Markdown parsers require it.